integration-libs/digital-payments/src/checkout/cms-components/dp-payment-method/dp-payment-form/dp-payment-form.module.ts

```diff
@@ -0,0 +1,23 @@
+import { DpCheckoutPaymentService } from '../../../facade/dp-checkout-payment.service';
+import { SpinnerModule } from '@spartacus/storefront';
+import { DpPaymentFormComponent } from './dp-payment-form.component';
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
+import { NgSelectModule } from '@ng-select/ng-select';
+import { I18nModule } from '@spartacus/core';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+    NgSelectModule,
+    I18nModule,
+    SpinnerModule,
+  ],
+
+  declarations: [DpPaymentFormComponent],
+  exports: [DpPaymentFormComponent],
+  providers: [DpCheckoutPaymentService],
```

Comment:
Is it needed? I think this service is providedIn root

Reply:
ya, have removed here